api_manager.py

```diff
@@ -1 +1,21 @@
-""" Manage API information. Get API information from individual API modules, send packaged results to user_interface """
\ No newline at end of file
+""" Manage API information. Get API information from individual API modules, send packaged results to user_interface """
+import tarot_api
+import animal_api
+import horoscope_api
+
```

Comment:
all variables should be in their own function, could be called "daily_mystic_fortune()"